api/envoy/config/core/v3/base.proto

```diff
@@ -461,11 +461,6 @@ message HeaderValueOption {
   // existing values. Otherwise it replaces any existing values.
   // This field is deprecated and please use
   // :ref:`append_action <envoy_v3_api_field_config.core.v3.HeaderValueOption.append_action>` as replacement.
-  //
-  // .. note::
-  //   The :ref:`external authorization service <envoy_v3_api_msg_service.auth.v3.CheckResponse>` and
```

Comment:
Checking ext_authz code, it had moved to take default append as true. So, this comments is stale.